lib/manageiq/appliance_console/cli.rb

```diff
@@ -232,6 +247,25 @@ def set_external_db
       config.post_activation
     end
 
+    def set_replication
+      if options[:replication] == "primary"
+        db_replication = ManageIQ::ApplianceConsole::DatabaseReplicationPrimary.new
+        say("Configuring Server as Primary")
+      else
+        db_replication = ManageIQ::ApplianceConsole::DatabaseReplicationStandby.new
+        say("Configuring Server as Standby")
+        db_replication.disk = disk_from_string(options[:dbdisk])
+        db_replication.primary_host = options[:primary_host]
+        db_replication.standby_host = options[:standby_host] unless options[:standby_host].nil?
```

Comment:
These are all double negatives, can these be `if options[:thing]` rather than `unless options[:thing].nil?`